javascript/javascript1/week1/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <title>Apply JavaScript example</title>
+    <script src="age-ify.js"></script>
+    <script src="dog-age.js"></script>
+    <script src="housey-pricey.js"></script>
+    <script src="Ez-namey.js"></script>
+    <script src="family-pizza.js"></script>
+    <script src="pizza-exercise.js"></script>
+
+
+
```

Comment:
It'd be nice to have something to look at when you enter the site.
I've seen some students adding images, and others adding a guide on how to open the console.

Reply:
I  couldn't understand what did you really mean here ?

Reply:
I mean that your page is index.html page is empty. 
You could put some text saying "Welcome to my page. Check the console to see the results"
or you could put a nice image :)